src/components/forms/SearchDropdown.tsx

```diff
@@ -46,16 +47,35 @@ export const SearchDropdown = ({ show = false, term, handleSearchClick, largeSpa
   const anchorClasses = (last: boolean) =>
     `flex flex-wrap items-center cursor-pointer py-2 px-4 block text-cpr-dark hover:bg-gray-200 focus:bg-gray-200 ${last ? "rounded-b-lg" : ""}`;
 
-  const renderSearchSuggestion = (geography: TGeography) => {
-    if (!term.toLowerCase().includes(geography.display_value.toLowerCase())) return;
-    if (!termWithoutGeography(geography.display_value).trim().length) return;
+  const renderSearchSuggestion = (geographies: TGeography[]) => {
+    if (!geographies || geographies == null || geographies.length === 0) return;
```

Comment:
Could you make the null check a `===`?

This could/should be a lint rule actually!

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness

Reply:
Oops, good spot! Will update.